_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -1055,9 +1015,12 @@ public static function update_settings_permission_check() {
 	/**
 	 * Verify that user can view Jetpack admin page and can activate plugins.
 	 *
-	 * @since 4.3.0
-	 *
 	 * @return bool Whether user has the capability 'jetpack_admin_page' and 'activate_plugins'.
+	 * @deprecated 8.8.0 The method is moved to the `REST_Connector` class.
+	 *
+	 * @see REST_Connector::activate_plugins_permission_check()
+	 *
+	 * @since 4.3.0
 	 */
 	public static function activate_plugins_permission_check() {
```

Comment:
Should we remove the implementation of this method and just call `REST_Connector::activate_plugins_permission_check()` in here?

Reply:
Good catch, fixed 👍